### ISSUE TITLE
Use natural numbers for the versions displayed in Collect

### ISSUE THREAD

**Issue by jnm** [kobotoolbox/kpi#3006]
Edit: ODK Collect [has changed their behavior](https://forum.getodk.org/t/issue-with-pulldata-update-of-form-you-have/32282) and now requires the `formList` version to match the XForm version. I think we'll comply with that by implementing what we'd already intended, but we need to verify that.

Replace the current unique ID shown in Collect with a string that includes the number displayed in "Form history" on the KPI form landing page.

The current behavior looks like this:

KPI "Form history" | Version shown in Collect
-- | --
v1 | Version: vw4Qx5WLYJreebXdAnyHdW

What we'd like to do instead, assuming that `v1` was deployed on 2021-02-10 11:58:03 UTC, is have:

KPI "Form history" | Version shown in Collect
-- | --
v1 | Version: 1 (2021-02-10 11:58:03)

There are two places in the Collect interface where we want the natural number version to appear:
1. In "Get Blank Forms": this is ~~(hopefully)~~ controlled by the `<version>` in the XML provided by `formList`
    * Tracked at kobotoolbox/kobocat#674
1. In "Fill Blank Forms": this is controlled by the `version` attribute on the first child element of `<instance>` inside the form XML (XForm) itself (see example XML below)
    * See https://github.com/kobotoolbox/kpi/blob/53cbd561d12d2b8d94fae83fb96048309d50a15e/kpi/models/asset.py#L424
    * As long as the `__version__` calculate question always contains the `AssetVersion.uid`, our data processing will continue to work correctly.

## Background information

From UNICEF:
- Would it be possible to to have a version number followed by the date. (i.e. 01 03082020) 
    - Yes, we can use any text string (as we’re using now). Maybe  
        - Version: 1 (deployed 2020-12-16) [Edit: probably should use a timestamp, as it's likely many versions could be deployed in a single day]

From John: 

- Collect - "fill blank form" shows version uid (starts with v) 
    - should be replaced with the sequential version number shown already in the KPI UI 
    - already shows "Added" date, but this is when the blank form was downloaded to the device 
    - should we show a deployment date alongside the sequential version number? 

- "get blank forms" 
    - right now it only shows the form id string (usually asset uid, starts with a) 
    - because we don't include a version in formList, but we can start 
    - should also show sequential version number to match "fill blank form" 
    - Collect will show version number alongside form id string

**Comment by jnm** [comment]
When this sample XML is served by https://kc.kobotoolbox.org/jnm_demo/forms/598398/form.xml, Collect shows `surprise` in the "Fill Blank Form" screen.
```xml
<?xml version="1.0" encoding="utf-8"?>
<h:html xmlns="http://www.w3.org/2002/xforms" xmlns:ev="http://www.w3.org/2001/xml-events" xmlns:h="http://www.w3.org/1999/xhtml" xmlns:jr="http://openrosa.org/javarosa" xmlns:odk="http://www.opendatakit.org/xforms" xmlns:orx="http://openrosa.org/xforms" xmlns:xsd="http://www.w3.org/2001/XMLSchema">
  <h:head>
    <h:title>I am a new name</h:title>
    <model>
      <instance>
        <aad8Fy7JuCZicw6rENbB4C id="aad8Fy7JuCZicw6rENbB4C" version="surprise">
          <formhub>
            <uuid/>
          </formhub>
          <start/>
          <end/>
          <I_am_a_question/>
          <__version__/>
          <meta>
            <instanceID/>
          </meta>
        </aad8Fy7JuCZicw6rENbB4C>
      </instance>
      <bind jr:preload="timestamp" jr:preloadParams="start" nodeset="/aad8Fy7JuCZicw6rENbB4C/start" type="dateTime"/>
      <bind jr:preload="timestamp" jr:preloadParams="end" nodeset="/aad8Fy7JuCZicw6rENbB4C/end" type="dateTime"/>
      <bind nodeset="/aad8Fy7JuCZicw6rENbB4C/I_am_a_question" required="false()" type="string"/>
      <bind calculate="'vezdSEXctjRryAgixLVrXu'" nodeset="/aad8Fy7JuCZicw6rENbB4C/__version__" type="string"/>
      <bind calculate="concat('uuid:', uuid())" nodeset="/aad8Fy7JuCZicw6rENbB4C/meta/instanceID" readonly="true()" type="string"/>
      <bind calculate="'5e698792a5f342beb081cdf6f0336eea'" nodeset="/aad8Fy7JuCZicw6rENbB4C/formhub/uuid" type="string"/>
    </model>
  </h:head>
  <h:body>
    <input ref="/aad8Fy7JuCZicw6rENbB4C/I_am_a_question">
      <label>I am a question</label>
    </input>
  </h:body>
</h:html>
```

**Comment by jnm** [comment]
Closed by #3046, #3081, kobotoolbox/kobocat#685
